ITISFoundation/osparc-simcore
# Adds MPI scheduling support

## What do these changes do?


Allow the user to schedule services on MPI nodes. Please check proposed implementation in the liked issue.

**Note:** In development mode, the `sidecar_mpi` service will be spawned as an MPI node. This is done by setting the `TARGET_MPI_NODE_CPU_COUNT` environment variable to the number of CPUs of the host.

## Related issue number



Closes #1672 
## How to test


Create a Study and add at least one copy of the following computational: `sidecar`, `sidecar-gpu` and `sidecar-mpi`.
Press play and check the logs of the sidecar_mpi swarm service.

## Checklist
